binderhub/tests/test_repoproviders.py

```diff
@@ -166,19 +167,24 @@ async def test_hydroshare_doi():
 
 
 @pytest.mark.parametrize(
+    # "10.7910/DVN/TJCLKP" is a DOI associated with all versions of the
+    # dataverse dataset, including the latest version and previous versions (v3,
+    # v2, etc). Dataverse doesn't mint DOIs for each version of a dataset, but
+    # that has been discussed in https://github.com/IQSS/dataverse/issues/4499,
+    #
     "spec,resolved_spec,resolved_ref,resolved_ref_url,build_slug",
     [
         [
             "10.7910/DVN/TJCLKP",
             "10.7910/DVN/TJCLKP",
-            "3035124.v3.0",
+            r"3035124.v\d+.\d+",
```

Comment:
```suggestion
            r"3035124\.v\d+\.\d+$",
```
`.` is a wildcard, `re.match` does a prefix match